src/main/java/reposense/report/CommitReportJson.java

```diff
@@ -6,20 +6,21 @@
 import reposense.authorship.model.AuthorshipSummary;
 import reposense.commits.model.AuthorIntervalContribution;
 import reposense.commits.model.CommitContributionSummary;
+import reposense.commits.model.CommitResult;
 import reposense.model.Author;
 
 /**
  * Class that holds the data to be serialized into JSON format in `commits.json`.
```

Comment:
You may want to also update the class header so it is consistent with our code style.

```Stores the data to be serialized...```

Reply:
As this is a class header and not a method header, I don't think there's a need to adjust the header to start with a verb.